gr-fec/include/gnuradio/fec/cc_encoder.h

```diff
@@ -103,7 +103,7 @@ class FEC_API cc_encoder : virtual public generic_encoder
      * \param frame_size Number of bits per frame. If using in the
      *        tagged stream style, this is the maximum allowable
      *        number of bits per frame.
-     * \param k Constraint length (K) of the encoder.
+     * \param k Constraint length (K) of the encoder. Maximum possible value is 32.
```

Comment:
Why this limitation? Can you add why to the comment, if it's brief?

Reply:
On second thoughts this is wrong, it must be 31: Constraint length K = 32 implies a polynomial with 32 coefficients. But the sign bit (a negative value) is used to determine whether to invert the output.